components/canvas/Cargo.toml

```diff
@@ -33,7 +33,7 @@ num-traits = "0.2"
 raqote = {git = "https://github.com/jrmuizel/raqote", optional = true}
 pixels = {path = "../pixels"}
 servo_config = {path = "../config"}
-sparkle = "0.1.8"
+sparkle = "0.1"
```

Comment:
Why? Lowering bounds in general is a mistake. You are stating the crate is compatible with `0.1.0` but it may require an API from `0.1.6`.

Reply:
This is only here for picking up the patch to Sparkle, if it lands I can update it to the version Sparkle will be at then.